runtimes/polimec/src/weights/pallet_preimage.rs

```diff
@@ -36,19 +35,19 @@ impl<T: frame_system::Config> pallet_preimage::WeightInfo for WeightInfo<T> {
 	/// Storage: `Preimage::RequestStatusFor` (r:1 w:1)
 	/// Proof: `Preimage::RequestStatusFor` (`max_values`: None, `max_size`: Some(91), added: 2566, mode: `MaxEncodedLen`)
 	/// Storage: `Balances::Holds` (r:1 w:1)
-	/// Proof: `Balances::Holds` (`max_values`: None, `max_size`: Some(949), added: 3424, mode: `MaxEncodedLen`)
+	/// Proof: `Balances::Holds` (`max_values`: None, `max_size`: Some(175), added: 2650, mode: `MaxEncodedLen`)
 	/// Storage: `Preimage::PreimageFor` (r:0 w:1)
 	/// Proof: `Preimage::PreimageFor` (`max_values`: None, `max_size`: Some(4194344), added: 4196819, mode: `MaxEncodedLen`)
 	/// The range of component `s` is `[0, 4194304]`.
 	fn note_preimage(s: u32, ) -> Weight {
 		// Proof Size summary in bytes:
-		//  Measured:  `135`
-		//  Estimated: `4414`
-		// Minimum execution time: 38_290_000 picoseconds.
-		Weight::from_parts(161_037_588, 0)
```

Comment:
Is this 5x increase normal?

Reply:
On the Collectives System Chain: https://github.com/polkadot-fellows/runtimes/blob/cf15699e1527f3bf4c76abd0287e62721bee428c/system-parachains/collectives/collectives-polkadot/src/weights/pallet_preimage.rs#L59-L70

It's definitely a strange increase, but the final weight is still ok.